Source/Project64/User Interface/Settings Config.cpp

```diff
@@ -88,6 +88,13 @@ LRESULT	CSettingConfig::OnInitDialog(UINT /*uMsg*/, WPARAM /*wParam*/, LPARAM /*
 
 	CConfigSettingSection * SettingsSection;
 
+	//Set the text for all gui Items
+	SetDlgItemTextW(m_hWnd, IDC_RESET_PAGE, GS(BOTTOM_RESET_PAGE));
+	SetDlgItemTextW(m_hWnd, IDC_RESET_ALL, GS(BOTTOM_RESET_ALL));
+	SetDlgItemTextW(m_hWnd, IDOK, GS(CHEAT_OK));

```

Comment:
I know this works ...I really should have just done a OK, CANCEL, APPLY .. since we now are applying a cheat string to a different code unrelated to cheat